bottles/frontend/ui/details-bottle.ui

```diff
@@ -118,12 +118,12 @@
                         <property name="valign">center</property>
                         <property name="halign">center</property>
                         <property name="margin-bottom">5</property>
-                        <property name="text" translatable="yes">Run with Arguments…</property>
+                        <property name="text" translatable="yes">Run with addition arguments…</property>
                     </object>
                 </child>
                 <child>
                     <object class="GtkCheckButton" id="check_terminal">
-                        <property name="label" translatable="yes">Launch in Terminal…</property>
+                        <property name="label" translatable="yes">Launch in a terminal…</property>
```

Comment:
```suggestion
                        <property name="label" translatable="yes">Launch in a Terminal…</property>
```

Reply:
Why the capitalization? I don't see why it would be needed.

Reply:
It's to keep it consistent with the dropdown menu and conforming with the GNOME HIG.